### PR TITLE
[docs] add simulation report page

### PR DESCRIPTION
**Pull request by MarineLeM**
Add simulation report page

### CODE REVIEW

File: docs/usage/simulation_reports.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Generating and Sharing Simulation Reports in PDF Format
+
+You can generate and share a simulation report in PDF format. 
+The following steps guide you through viewing, creating, and editing simulation reports, as well as generating PDF
+of these reports.
+
+## Viewing the List of Simulation Reports
+* Access the Simulation Page
+* Open the Report Drawer: Click the "Edit" button located on the right side of the page, then select "Access Reports"
```
Comment (EllynBsc) [inline docs/usage/simulation_reports.md:9]:
```suggestion
* Open the Report Drawer: Click on the "Edit" button located on the right side of the page, then select "Access Reports"
```

File: docs/usage/simulation_reports.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+![simulation_edit_button](./assets/simulation_reports/simulation_edit_button.png)
+
+* A drawer will open, displaying the list of all previously generated reports for the selected simulation.
+* Click on the report name from the list to view the full report details.
+
+![simulation_report_page](./assets/simulation_reports/simulation_report_page.png)
+ 
+
+## Creating a New Simulation Reports
+1. At the bottom of the report list in the drawer, click the "Add" button to create a new report.
```
Comment (EllynBsc) [inline docs/usage/simulation_reports.md:20]:
```suggestion
1. At the bottom of the report list in the drawer, click on the "Add" button to create a new report.
```

**Review by EllynBsc — CHANGES_REQUESTED** [review]
Hi @MarineLeM 👋 

Nice! Very clear, tiny english idioms but LGTM 🚀

**Review by EllynBsc — APPROVED** [review]

**Comment by MarineLeM** [comment]
@EllynBsc ready for a second round

**Pull request merged**